.gitmodules

```diff
@@ -16,7 +16,7 @@
 [submodule "BaseTools/Source/C/BrotliCompress/brotli"]
 	path = BaseTools/Source/C/BrotliCompress/brotli
 	url = https://github.com/google/brotli
-	ignore = untracked
+	ignore = untracked
```

Comment:
what is this change?

Reply:
line endings were not consistent in file.